test.sh

```diff
@@ -98,4 +98,4 @@ docker run --network=test-network -v $(pwd):/repos --workdir /repos/ -e INSTALL_
 echo "INFO: Running the tests for database schema version 0.0.1"
 docker exec mysql sh -c 'mysql -u root -pP@ssw0rd < /scripts/test-queries/2-test-0.0.1.sql' > log.txt
 errors=$(cat log.txt | grep "^Error" || true)
-if [ -n "$errors" ]; then echo $errors && exit 1; fi
+if [ -n "$errors" ]; then echo $errors && exit 1; fi
```

Comment:
Why did you change the test file? Return back EOF
![image](https://github.com/mate-academy/rda_task_9_running_database_migrations/assets/87720583/48469057-dfc9-4d33-8408-49d57d9f09ba)